util/github/common.sh

```diff
@@ -44,7 +44,7 @@ install_macosx_deps() {
 }
 
 # Relative to git-repository root:
-TRIGGER_COMPILE_PATHS="src/.*\.(c|cpp|h)|CMakeLists.txt|cmake/Modules/|util/travis/|util/buildbot/"
+TRIGGER_COMPILE_PATHS="src/.*\.(c|cpp|h)|CMakeLists.txt|cmake/Modules/|util/github/|util/buildbot/"
 
 needs_compile() {
 	RANGE="$TRAVIS_COMMIT_RANGE"
```

Comment:
nowhere i think, it's just a sed

Reply:
This check has a point though, it should not just be removed.

Before starting the CI process this function would be called to determine if compiling was necessary, so that C++ compiling would be skipped if you e.g. change something in `builtin/`.
Is it possible to do the same thing with Github actions?

Reply:
yep we can filter actions depending on what we modified, like in gitlab

Reply:
Does it do that yet?

Reply:
not yet but we can add this later it's minor as the builds are far more faster than travis. In 5 minutes you have your result with actions whereas before it tooks 30 mins...

Reply:
it seems we can filter by having multiple workflows, i will try to split one of the linter to see if we can trigger it properly